components/places/src/api/mod.rs

```diff
@@ -10,5 +10,6 @@ use observation::{VisitObservation};
 use storage;
 
 pub fn apply_observation(conn: &mut PlacesDb, visit_obs: VisitObservation) -> Result<()> {
```

Comment:
I'm not sure it makes sense to have this wrapper TBH, but it can stay for now.

Reply:
I'm now using this in the ffi so it doesn't need to deal with the RowId